homeassistant/components/yeelight/light.py

```diff
@@ -720,6 +732,7 @@ async def async_set_flash(self, flash) -> None:
             count = 1
             duration = transition * 2
 
+        assert self.hs_color
```

Comment:
`hs_color` can be `None`; this should not be asserted.

See:
<img width="427" alt="CleanShot 2022-04-03 at 22 30 44@2x" src="https://user-images.githubusercontent.com/195327/161447295-55124769-3824-4d7f-bf14-7ba808c61dc5.png">

Reply:
`mypy` complains because `color_util.color_hs_to_RGB` doesn't accept `None`

I have adjusted it to return early here, but maybe there is a better way to handle it...
```python
        if int(self._bulb.last_properties["color_mode"]) != 1 or not self.hs_color:
            _LOGGER.error("Flash supported currently only in RGB mode")
            return
```